info.xml

```diff
@@ -18,7 +18,7 @@
   <version>1.4-dev</version>
   <develStage>dev</develStage>
   <compatibility>
-    <ver>5.45</ver>
+    <ver>5.68</ver>
```

Comment:
This is the CiviCRM version that includes `\CRM_Utils_ICalendar::createCalendarFileForEvent()` and subsequent regression fixes, see civicrm/civicrm-core#26980.